.github/prompts/wxpython-specialist.prompt.md

```
@@ -0,0 +1,89 @@
+agent: wxpython-specialist
+tools:
+
+- askQuestions
+- readFile
+- editFiles
+- createFile
+- listDirectory
+- runInTerminal
+- getTerminalOutput
+
+______________________________________________________________________
```

markdownlint-cli2 (0.21.0)

[warning] Horizontal rule style
Expected: ---; Actual: ______________________________________________________________________

(MD035, hr-style)